src/map/clif.c

```diff
@@ -20482,6 +20622,13 @@ void do_final_clif(void)
 			aFree(clif->rd.qty[i]);
 	}
 
+	for (i = 0; i < MAX_ATTENDANCE_DAYS; i++) {
+		if (clif->attendance_data.nameid[i])
+			aFree(clif->attendance_data.nameid[i]);
```

Comment:
you cant remove int values.
if need clean id's set array fields to 0.

same for second aFree.